awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -67,8 +76,28 @@ class WorkflowJobTemplate extends Component {
         );
         data.summary_fields.webhook_credential.kind = name;
       }
+      const [notifAdminRes, auditorRes, adminRes] = await Promise.all([
+        OrganizationsAPI.read({
+          page_size: 1,
+          role_level: 'notification_admin_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'auditor_role',
+        }),
+        OrganizationsAPI.read({
+          id: data.organization,
+          role_level: 'admin_role',
+        }),
```

Comment:
I don't think we need all three of these requests for WFJTS. On [Template.jsx](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Template/Template.jsx#L53) , it looks  like we only request `notification_admin_role`.

I only see one request to `/api/organizations/...` in the network tab when loading a WFJT and clicking into notifications in the current app.

Reply:
@jlmitch5 I think I included these requests in the Project.jsx as well but I don't think they're needed for any view except Orgs.  We should either rip them out of Project.jsx too or create an issue to do so outside of this PR.